snap7/util.py

```diff
@@ -270,6 +274,59 @@ def set_dint(_bytearray, byte_index, dint):
     for i, b in enumerate(_bytes):
         _bytearray[byte_index + i] = b
 
+def get_s5time(_bytearray, byte_index):
+    micro_to_milli = 1000
+    data_bytearray = _bytearray[byte_index:byte_index+2]
+    S5Time_data_int_like = list(data_bytearray.hex())
+    if S5Time_data_int_like[0]=='0':
+        # 10ms
+        time_base = 10
+    elif S5Time_data_int_like[0]=='1':
+        # 100ms
+        time_base = 100
+    elif S5Time_data_int_like[0]=='2':
+        # 1s
+        time_base = 1000
+    elif S5Time_data_int_like[0]=='3':
+        # 10s
+        time_base = 10000
+    else:
+        raise ValueError
+    print('This value should not be greater than 3')
```

Comment:
Change this into a log.debug, prints are ok if you debug for yourself or realy need a print().

Reply:
Hello,
I have tested my code in step7 on simulation mode through nettoplcsim. 
I only have a soft plc. This typ is WinLC RTX F. I think, this will also work on real hard plc cpu.

Thank you for your suggestion, I'll change the print into log.debug.